contracts/interfaces/IMultiwrap.sol

```diff
@@ -4,28 +4,23 @@ pragma solidity ^0.8.11;
 import "./IThirdwebContract.sol";
 import "./IThirdwebRoyalty.sol";
 import "./IThirdwebOwnable.sol";
+import "../lib/MultiTokenTransferLib.sol";
 
 interface IMultiwrap is IThirdwebContract, IThirdwebOwnable, IThirdwebRoyalty {
```

Comment:
I think we should keep `MultiToken` struct in the `IMultiwrap`
logically, if I want to build on top of multiwrap, all i'd need is IMultiwrap and it has all it's relevant struct.